LogMonitor/src/LogMonitor/Utility.cpp

```diff
@@ -269,12 +269,16 @@ void Utility::SanitizeJson(_Inout_ std::wstring& str)
     while (i < str.size()) {
         auto sub = str.substr(i, 1);

```

Comment:
There is no block to handle if the incoming _Inout_ std::wstring& str parameter only has a single character like "~". We should probably remove that character.

Reply:
Also is there a case where the parameter is empty and how are we handling that?

Reply:
This has been fixed. Please re-check.